Frameworks/CoreGraphics/CGPattern.mm

```diff
@@ -198,3 +209,24 @@ CGPatternRef CGPatternRetain(CGPatternRef pattern) {
     CFRetain((id)pattern);
     return pattern;
 }
+
+CGRect _CGPaternGetBounds(CGPatternRef pattern) {
+    RETURN_RESULT_IF_NULL(pattern, CGRectNull);
+    return ((CGPattern*)pattern)->bounds;
+}
+
+void _CGPaternIssueCallBack(CGContextRef context, CGPatternRef pattern) {
+    FAIL_FAST_IF_NULL(context);
+    FAIL_FAST_IF_NULL(pattern);
+    ((CGPattern*)pattern)->callbacks.drawPattern(((CGPattern*)pattern)->info, context);
+}
+
+CGAffineTransform _CGPatternGetTransformation(CGPatternRef pattern) {
+    RETURN_RESULT_IF_NULL(pattern, CGAffineTransformIdentity);
+    return ((CGPattern*)pattern)->matrix;
+}
+
+CGRect _CGPatternGetFinalPatternSize(CGPatternRef pattern) {
+    RETURN_RESULT_IF_NULL(pattern, CGRectNull);
```

Comment:
nope,. this should never happen. this should cause a failure

---
In reply to: [93077871](https://github.com/Microsoft/WinObjC/pull/1586#discussion_r93077871) [](ancestors = 93077871)